packages/kbn-std/src/rxjs_7.ts

```diff
@@ -6,14 +6,4 @@
  * Side Public License, v 1.
  */
 
-import { Observable } from 'rxjs';
-import { first, last } from 'rxjs/operators';
-
-export function firstValueFrom<T>(source: Observable<T>) {
-  // we can't use SafeSubscriber the same way that RxJS 7 does, so instead we
-  return source.pipe(first()).toPromise();
-}
-
-export function lastValueFrom<T>(source: Observable<T>) {
-  return source.pipe(last()).toPromise();
-}
+export { firstValueFrom, lastValueFrom } from 'rxjs';
```

Comment:
We should probably get rid of this file and adapt usages instead.

Reply:
done 👍